doc/api/fs.md

```diff
@@ -4377,10 +4377,6 @@ the returned {fs.FSWatcher}.
 The `fs.watch` API is not 100% consistent across platforms, and is
 unavailable in some situations.
```

Comment:
Is it still true that it is unavailable in some situations?

Reply:
IBMi and AIX does not fully support `fs.watch`

Reply:
As a task for later (follow on PR) it would be good to explain a bit more what "situations" it is unavailable.